### PR TITLE
Add username, parent pid, parent process name

### PR DESCRIPTION
**Pull request by mew1033**

### CODE REVIEW

File: hubblestack_nebula/hubblestack_nebula_queries.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
   - query_name: listening_procs
-    query: SELECT t.unix_time AS query_time, h.md5 AS md5, p.pid AS process_id, name AS process, ltrim(address, ':f') AS address, port AS dest_port, p.path AS file_path, cmdline, p.on_disk, root, parent, CASE lp.protocol WHEN 6 THEN 'tcp' WHEN 17 THEN 'udp' ELSE lp.protocol END as transport FROM listening_ports AS lp LEFT JOIN processes AS p ON lp.pid=p.pid LEFT JOIN time AS t LEFT JOIN hash AS h ON h.path=p.path;
+    query: SELECT t.unix_time AS query_time, h.md5 AS md5, p.pid AS process_id, p.name AS process, ltrim(address, ':f') AS address, port AS dest_port, p.path AS file_path, p.cmdline, p.on_disk, p.root, p.parent, pp.name AS parent_process, CASE lp.protocol WHEN 6 THEN 'tcp' WHEN 17 THEN 'udp' ELSE lp.protocol END as transport FROM listening_ports AS lp LEFT JOIN processes AS p ON lp.pid=p.pid LEFT JOIN processes AS pp ON p.parent=pp.pid LEFT JOIN time AS t LEFT JOIN hash AS h ON h.path=p.path;
```
Comment (basepi) [inline hubblestack_nebula/hubblestack_nebula_queries.yaml]: There are a couple of pp. instances in this line. Are those correct? Just wanted to double check.

Comment (basepi) [reply hubblestack_nebula/hubblestack_nebula_queries.yaml]: Oh, I just spotted the left join. Probably fine.

File: hubblestack_nebula/hubblestack_nebula_win_queries.yaml [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
   - query_name: win_listening_procs
-    query: SELECT t.unix_time AS query_time, h.md5 AS md5, p.pid AS process_id, name AS process, ltrim(address, ':f') AS address, port AS dest_port, p.path AS file_path, cmdline, p.on_disk, root, parent, CASE lp.protocol WHEN 6 THEN 'tcp' WHEN 17 THEN 'udp' ELSE lp.protocol END as transport FROM listening_ports AS lp LEFT JOIN processes AS p ON lp.pid=p.pid LEFT JOIN time AS t LEFT JOIN hash AS h ON h.path=p.path;
+    query: SELECT t.unix_time AS query_time, h.md5 AS md5, p.pid AS process_id, p.name AS process, ltrim(address, ':f') AS address, port AS dest_port, p.path AS file_path, p.cmdline, p.on_disk, p.root, p.parent, pp.name AS parent_process, CASE lp.protocol WHEN 6 THEN 'tcp' WHEN 17 THEN 'udp' ELSE lp.protocol END as transport FROM listening_ports AS lp LEFT JOIN processes AS p ON lp.pid=p.pid LEFT JOIN processes AS ppp ON p.parent=pp.pid LEFT JOIN time AS t LEFT JOIN hash AS h ON h.path=p.path;
```
Comment (basepi) [inline hubblestack_nebula/hubblestack_nebula_win_queries.yaml]: This one has references to `pp` but only a left join as `ppp`. Am I missing something?

Comment (mew1033) [reply hubblestack_nebula/hubblestack_nebula_win_queries.yaml]: Oops, my bad. Fixed it.

**Pull request merged**